firefox-ios/Client/Application/main.swift

```diff
@@ -4,7 +4,7 @@
 
 import Common
 
-private var appDelegate: String?
+private let appDelegate: String
 
```

Comment:
Ahh! That's the change I saw in https://github.com/mozilla-mobile/firefox-ios/pull/27435 👀 👍